python-test/features/steps/functions.py

```diff
@@ -0,0 +1,77 @@
+import requests
+import json
+# from pygments import highlight, lexers, formatters
+import os
+import docker
+import re
+
+base_orb_url = os.getenv('ORB_URL', "https://beta.orb.live")
+
+
+def generate_token(email, password, base_orb_url = base_orb_url):
+    if email != None and password != None: 
+        headers={'Content-type':'application/json',  'Accept':'*/*'}
+        token_request = requests.post(base_orb_url+'/api/v1/tokens', json= {'email': email, 'password': password}, headers=headers)
+        if token_request.status_code >= 200 and token_request.status_code <300:
+            return (((token_request.text).split(":"))[1]).split('"')[1]
```

Comment:
Is the response a JSON string? Should we parse it instead of manipulating the string?

Reply:
agreed. `requests` library makes this easy, just use `token_request.json()` instead of `.text`

https://docs.python-requests.org/en/latest/user/quickstart/#json-response-content